### DISCUSSION TITLE
[제안] 프로젝트 테이블에 'Priority' 필드 추가의 건

### DISCUSSION THREAD

**Discussion by DragonTiger92** [FRONTENDSCHOOL8/Book-Kong#89] [Ideas]
프로젝트 테이블(우리가 '백로그'라고 부르는 거)에서 라벨을 보는데, 라벨과 Priority 정보가 섞여 있으니 보기에 조금 불편하더라고요. 'Priority' 필드를 만들어 거기에서 Priority 필드의 값을 'Critical', 'High', 'Medium', 'Low' 중 하나로 고를 수 있게 설정하면 조금 더 테이블을 편하게 볼 수 있을 것 같은데...... 어떻게 생각하시나요??????

**Comment by Samuel-0930** [comment]
저는 Priority가 있는게 이 라벨은 중요도에 관련된 거구나 하고 좀 더 쉽게 인지가 되는 것 같아서 있는게 좋을 것 같습니다!

**Reply by DragonTiger92** [reply]
아, 라벨에서 Priority를 떼자는 게 아니라 아예 Labels 말고 'Priority'라는 필드를 따로 만들어서 그 안에서 priority 정보만 따로 관리하자는 말이었는데...... 무튼 지금 딱히 중요한 부분은 아니니 그냥 넘어가도 될 것 같습니다. 의견 주셔서 감사합니다!

**Comment by DragonTiger92** [comment]
현 개발 단계에서 딱히 중요한 issue는 아닌 것 같아서 임의로 discussion을 종료함.
